### PR TITLE
delegating `max_query_parameters` to katsu, added default translation setting

### PR DESCRIPTION
**Pull request by brouillette**
Integrating changes from [this pr](https://github.com/bento-platform/bento_public/pull/75) into the main branch

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -123,21 +123,70 @@ func main() {
...
 
 		return c.JSON(http.StatusOK, rf(jsonLike))
 	}
+
 	katsuRequestBasic := func(path string, c echo.Context) error {
 		return katsuRequest(path, nil, c, identityJSONTransform)
 	}
 
+	fetchAndSetKatsuPublic := func(c echo.Context, katsuCache *cache.Cache) (JsonLike, error) {
+		fmt.Println("'publicOverview' not found or expired in 'katsuCache - fetching")
```
Comment (davidlougheed) [inline main.go:153]: minor - missing closing single quote after katsuCache

**Review by davidlougheed — CHANGES_REQUESTED** [review]
one minor change. also, can you rename the PR from a version to a description of what this PR is doing and undraft it? we may or may not make this v0.12 depending on if we also get Beacon UI merged.

**Review by davidlougheed — APPROVED** [review]
lgtm

**Review by gsfk — COMMENTED** [review]
No issues for me, other than a small merge conflict with my own PR, which shouldn't be to hard to manage.

**Pull request merged**